src/mock-json/example3.json

```diff
@@ -0,0 +1,129 @@
+{"eventLog1": [
+  {
+    "00000000": [
+        [
+            "prefix_1:\"First outpatient consultation\"",
+            0.2612
+        ],
+        [
+            "prefix_2:\"administrative fee - the first pol\"",
+            0.1501
+        ],
+        [
+            "prefix_3:\"verlosk.-gynaec. short-out card cost\"",
+            -0.0105
+        ],
+        [
+            "prefix_4:\"ultrasound - internal genitals\"",
+            0.0027
+        ],
+        [
+            "prefix_5:\"First outpatient consultation\"",
+            -0.0003
+        ],
+        [
+            "prefix_6:\"administrative fee - the first pol\"",
+            -0.0105
+        ],
+        [
+            "prefix_7:\"simulator - use before megavol\"",
+            0.0025
+        ],
+        [
+            "prefix_8:\"treatment time - Unit t3 - megavolt\"",
+            -0.0033
+        ],
+        [
+            "prefix_9:\"teletherapy - megavolt photons bestrali\"",
+            -0.1
+        ]
+    ]
+  }
+],
+"eventLog2":
+[
+  {
+    "00000003": [
+      [
+        "prefix_1:\"inwend.geneesk. short-out card cost\"",
+        0.2612
+      ],
+      [
+        "prefix_2:\"assumption laboratory\"",
+        0.1501
+      ],
+      [
+        "prefix_3:\"urine analysis - qualitative\"",
+        -0.005
+      ],
+      [
+        "prefix_4:\"order rate\"",
+        0.007
+      ],
+      [
+        "prefix_5:\"outpatient follow-up consultation\"",
+        -0.003
+      ],
+      [
+        "prefix_6:\"administrative fee - the first pol\"",
+        -0.01
+      ],
+      [
+        "prefix_7:\"outpatient follow-up consultation\"",
+        0.0025
+      ],
+      [
+        "prefix_8:\"outpatient follow-up consultation\"",
+        -0.033
+      ],
+      [
+        "prefix_9:\"inwend.geneesk. aanv.kaart cost-Out\"",
+        -0.1
+      ]
+    ]
+  }
+],
+"eventLog3":
+[
+  {
+    "00000007": [
+      [
+        "prefix_1:\"verlosk.-gynaec. short-out card cost\"",
+        0.2612
+      ],
+      [
+        "prefix_2:\"histological examination - biopsies nno\"",
+        0.1501
+      ],
+      [
+        "prefix_3:\"assumption laboratory\"",
+        -0.005
+      ],
+      [
+        "prefix_4:\"assumption laboratory\"",
+        0.007
+      ],
+      [
+        "prefix_5:\"assumption laboratory\"",
+        -0.003
+      ],
+      [
+        "prefix_6:\"unconjugated bilirubin\"",
+        -0.01
+      ],
+      [
+        "prefix_7:\"bilirubin - total\"",
+        0.0025
+      ],
+      [
+        "prefix_8:\"glucose\"",
+        -0.033
+      ],
+      [
+        "prefix_9:\"urea\"",
+        -0.1
+      ]
+    ]
+  }
+]
+}
```

Comment:
the src folder it is supposed to contain only code, move this outside the src folder